libraries/AP_Scripting/examples/quadruped.lua

```diff
@@ -125,55 +126,55 @@ end
 -- specific order and this is repeated over and over to attain linear motion. when a
 -- specific leg number is passed the update_leg() produces the set of values for the
 -- given leg at that step, for each cycle of the gait each leg will move to a set
--- distance which is decided by the x_speed, yaw_speed, y_speed
+-- distance which is decided by the x_travel, yaw_travel, y_travel
 function update_leg(moving_leg)
     local leg_step = gait_step - gait_step_leg_start[moving_leg]
-    local move_requested = (math.abs(x_speed) > speed_dz) or (math.abs(y_speed) > speed_dz) or (math.abs(yaw_speed) > speed_dz)
+    local move_requested = (math.abs(x_travel) > travel_dz) or (math.abs(y_travel) > travel_dz) or (math.abs(yaw_travel) > travel_dz)
```

Comment:
Again just pass this in as an argument from above, would have saved you a lot of editing on this line :)

Reply:
changed as an argument